### PR TITLE
DDF-1305: Address all high severity CLM vulnerabilities

### PR DESCRIPTION
**Pull request by ghost**
- Changed versions containing embedded jars with security vulnerabilities.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf-parent/29)

### CODE REVIEW

**Comment by ghost** [comment]
@vinamartin @figliold @lessarderic

File: pom.xml [hunk 1]
```diff
@@ -394,7 +394,7 @@
                 <plugin>
                     <groupId>org.apache.karaf.tooling</groupId>
                     <artifactId>features-maven-plugin</artifactId>
-                    <version>2.3.8</version>
+                    <version>2.4.2</version>
```
Comment (lessarderic) [inline pom.xml]: The other PR (https://github.com/codice/ddf/pull/66) was using 2.4.1. We should be consistent and use 2.4.2 everywhere if that's the version that addresses all the vulnerabilities.

Comment (ghost) [inline pom.xml]: Fixed in other PR

**Comment by lessarderic** [comment]
:+1:

**Comment by figliold** [comment]
:+1: 

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf-parent/29)

**Comment by shaundmorris** [comment]
Change looks good to use current karaf 2.4.1 as the version. Full build and test was good.

**Pull request merged**